src/renderer/vsync/mod.rs

```diff
@@ -4,59 +4,43 @@ mod macos_display_link;
 mod vsync_macos;
 mod vsync_timer;
 #[cfg(target_os = "windows")]
-mod vsync_win;
+mod vsync_win_dwm;
 
 use vsync_timer::VSyncTimer;
 
 use crate::{
-    renderer::WindowedContext, settings::SETTINGS, window::UserEvent, window::WindowSettings,
+    renderer::SkiaRenderer, settings::SETTINGS, window::UserEvent, window::WindowSettings,
 };
-use winit::event_loop::EventLoopProxy;
+use winit::{event_loop::EventLoopProxy, window::Window};
 
 #[cfg(target_os = "linux")]
 use std::env;
```

Comment:
Fwiw due to the moved detection logic, this import is actually unused and yields a warning

```rust
warning: unused import: `std::env`
  --> src/renderer/vsync/mod.rs:17:5
   |
17 | use std::env;
   |     ^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default
```

Reply:
That is now fixed. I was doing most of my testing on Windows, and that was part of one late change that never got properly tested on Linux.